### PR TITLE
Feature/priya week3

### PR DESCRIPTION
**Pull request by Priyadharshini-842105**
Please review week3 assignment

### CODE REVIEW

File: assignment/src/com/assignment3/fileInputOutput/Main.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        //main class object
+        InputAndOutputStreams streams =new InputAndOutputStreams();
+        //assigning the methods
+        streams.filesExchange();
```
Comment (VBhaskarachary) [inline assignment/src/com/assignment3/fileInputOutput/Main.java:11]: Need  to display content of Target file.

File: assignment/src/com/assignment3/fileInputOutput/InputAndOutputStreams.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        System.out.println("Contents of the file: "+buffer);
+        //file object to hold the destination file
+        File dest=new File("C:\\Users\\842105\\Documents\\targetFile.txt");
+        //instantiating FileWriter object
+        FileWriter writer=new FileWriter(dest);
+        //writing contents to destination file
+        writer.write(buffer.toString());
+        writer.flush();
+        System.out.println("File copied successfully!!");
+
```
Comment (VBhaskarachary) [inline assignment/src/com/assignment3/fileInputOutput/InputAndOutputStreams.java:47]: inputStream and writer should be closed in finally block

File: assignment/src/com/assignment3/nullException/Main.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+public class Main {
+
+    public static void main(String[] args) {
+        //main class object
+        ExceptionHandling obj=new ExceptionHandling();
+        //assigning the methods
+        obj.divideByZero();
+        obj.nullPointer();
+    }
```
Comment (VBhaskarachary) [inline assignment/src/com/assignment3/nullException/Main.java:10]: Need to multiple exception handling within one method for both exception

File: assignment/src/com/assignment3/fileInputOutput/Main.java (hunk 1 above)
Comment (Priyadharshini-842105) [reply assignment/src/com/assignment3/fileInputOutput/Main.java:11]: fix

File: assignment/src/com/assignment3/fileInputOutput/InputAndOutputStreams.java (hunk 2 above)
Comment (Priyadharshini-842105) [reply assignment/src/com/assignment3/fileInputOutput/InputAndOutputStreams.java:47]: fix

File: assignment/src/com/assignment3/nullException/Main.java (hunk 3 above)
Comment (Priyadharshini-842105) [reply assignment/src/com/assignment3/nullException/Main.java:10]: fix

**Review by VBhaskarachary — APPROVED** [review]
changes looks good to me

**Pull request merged**